lessons/basics/sigils.md

```diff
@@ -24,6 +24,7 @@ A list of available sigils include:
   - `~s` Generates a string **with** escaping and interpolation
   - `~W` Generates a word list **with no** escaping or interpolation
   - `~w` Generates a word list **with** escaping and interpolation
+  - `~N` Generates a `NativeDateTime` struct
```

Comment:
I suppose it's [`Naive`](http://elixir-lang.org/docs/stable/elixir/NaiveDateTime.html), not `Native`.